Frontend/Gmfctn/src/app/modules/dashboard/exoft-achievements/exoft-achievements.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import '../../../models/achievement';
+import '../../../models/user';
+import { SaythankModalComponent } from '../../modal-windows/saythank-modal/saythank-modal.component';
+@Component({
+  selector: 'app-exoft-achievements',
+  templateUrl: './exoft-achievements.component.html',
+  styleUrls: ['./exoft-achievements.component.scss', '../../../app.component.scss']
```

Comment:
Why you import styles from app.components.scss? 
If you have some common styles there - just place that style in styles.scss and it will be used in every place where you use that appropriate class/id/tag